tower-grpc-build/src/client.rs

```diff
@@ -95,6 +100,11 @@ impl ServiceGenerator {
                 .bound("T::ResponseBody", "grpc::Body")
                 .arg_mut_self()
                 .line(format!("let path = http::PathAndQuery::from_static({});", path))
+                .doc(&method.comments.leading
```

Comment:
Take it or leave it, but it _might_ be worth factoring out the repeated code into a helper function? That way, we can just say things like
```rust
// ...
.doc(comments_to_rustdoc(&method.comments))
// ...
```
or something like that, here and elsewhere?

This would also have the advantage that if we were to change this logic later (such as to add the `leading_detached` comments, for example) we would only need to make the change in one place.

Reply:
I'm happy to make this change if you would like to wait to merge it. I appreciate the feedback!

Reply:
It's up to you! We can also merge this now and make that change in a follow-up PR, if you prefer. 

Just let me know if you're planning on addressing it in this branch; otherwise I'll go ahead and merge the PR.

Reply:
I'll address it in this branch. I'll push up those changes soon.